<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>potrace GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewZoomIn"/>
    <addaction name="actionViewZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionViewZoomReset"/>
    <addaction name="separator"/>
    <addaction name="actionViewRefresh"/>
    <addaction name="actionViewAutoRefresh"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSaveAs"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewZoomIn"/>
   <addaction name="actionViewZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionViewZoomReset"/>
   <addaction name="separator"/>
   <addaction name="actionViewRefresh"/>
   <addaction name="actionViewAutoRefresh"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_7">
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32768</width>
     <height>32768</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Potrace options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Advanced</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_54">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-z, --turnpolicy</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="ptTurnpolicy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>4</number>
          </property>
          <item>
           <property name="text">
            <string>black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>white</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>minority</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>majority</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>random</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_55">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-t, --turdsize</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="ptTurdsize">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_56">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-a, --alphamax</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="ptAlphamax">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="ptLongcurve">
          <property name="text">
           <string>-n, --longcurve</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="ptOpttolerance">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_57">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-O, --opttolerance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_58">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-u, --unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="ptUnit">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Frontend and Color</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="2" column="1">
         <widget class="QCheckBox" name="ptOpaque">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>--opaque</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_32">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-k, --blacklevel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="ptBlacklevel">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="ptInvert">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-i, --invert</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Scaling and Placement</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_47">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-P, --pagesize</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="ptPagesize">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>A4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Letter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Legal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tabloid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Statement</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Executive</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Folio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quarto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10x14</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="ptTight">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>--tight</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>-1</number>
       </property>
       <item row="0" column="1">
        <widget class="QComboBox" name="ptBackend">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>eps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pdf</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>svg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dxf</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>geojson</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pgm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gimppath</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>xfig</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>BACKEND TYPES</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAs">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>SaveAs...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/ICONS/res/picture_save.png</normaloff>:/ICONS/res/picture_save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>128</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Quick Convert</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewPreview"/>
  </widget>
  <action name="actionFileClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/folder.png</normaloff>:/ICONS/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionViewZoomIn">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/zoom_in.png</normaloff>:/ICONS/res/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>ZoomIn</string>
   </property>
   <property name="statusTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionViewZoomOut">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/zoom_out.png</normaloff>:/ICONS/res/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="statusTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionViewZoomReset">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/zoom.png</normaloff>:/ICONS/res/zoom.png</iconset>
   </property>
   <property name="text">
    <string>ZoomReset</string>
   </property>
   <property name="statusTip">
    <string>Zoom Reset</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/disk.png</normaloff>:/ICONS/res/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/disk_multiple.png</normaloff>:/ICONS/res/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs...</string>
   </property>
   <property name="toolTip">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/information.png</normaloff>:/ICONS/res/information.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionViewRefresh">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/arrow_refresh.png</normaloff>:/ICONS/res/arrow_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionViewAutoRefresh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/picture_link.png</normaloff>:/ICONS/res/picture_link.png</iconset>
   </property>
   <property name="text">
    <string>AutoRefresh</string>
   </property>
   <property name="statusTip">
    <string>AutoRefresh</string>
   </property>
  </action>
  <action name="actionPreviewHori">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/application_tile_horizontal.png</normaloff>:/ICONS/res/application_tile_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>PreviewHori</string>
   </property>
  </action>
  <action name="actionPreviewVirt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/ICONS/res/application_tile_vertical.png</normaloff>:/ICONS/res/application_tile_vertical.png</iconset>
   </property>
   <property name="text">
    <string>PreviewVirt</string>
   </property>
  </action>
  <action name="actionViewPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ICONS/res/picture_edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="toolTip">
    <string>Preview</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btnSaveAs</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>actionFileSaveAs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
